### ISSUE TITLE
cinnamon-menu-editor does not start

### ISSUE THREAD

**Issue by msjr65** [linuxmint/cinnamon#8965]
```
 * Cinnamon version (cinnamon --version)
Cinnamon 4.2.4
   - Please specify if you are using the daily builds PPA (https://launchpad.net/~linuxmint-daily-build-team/+archive/ubuntu/daily-builds).
NO
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...)
Arch
 * Graphics hardware *and* driver used
Nvidia G440 / nvidia-390xx  390.132-1
 * 32 or 64 bit
64 bit
 * Attach /home/<username>/.xsession-errors, or /var/log/syslog
 `
[xsession-errors.tar.gz](https://github.com/linuxmint/cinnamon/files/3856723/xsession-errors.tar.gz)

``

**Issue**
cinnamon-menu-editor does not start


**Steps to reproduce**
[xxxxxxxx ~]$ cinnamon-menu-editor
Traceback (most recent call last):
  File "/usr/bin/cinnamon-menu-editor", line 29, in <module>
    main()
  File "/usr/bin/cinnamon-menu-editor", line 24, in main
    app.run()
  File "/usr/share/cinnamon/cinnamon-menu-editor/cme/MainWindow.py", line 66, in run
    self.loadMenus()
  File "/usr/share/cinnamon/cinnamon-menu-editor/cme/MainWindow.py", line 201, in loadMenus
    self.loadMenu({ None: None })
  File "/usr/share/cinnamon/cinnamon-menu-editor/cme/MainWindow.py", line 212, in loadMenu
    name = cgi.escape(menu.get_name())
AttributeError: module 'cgi' has no attribute 'escape'


**Expected behaviour**
cinnamon editor should open


**Other information**

**Comment by icarter09** [comment]
@msjr65 what happens when you run `cinnamon-menu-editor` from the terminal?

**Comment by msjr65** [comment]
This is what I have shown under steps to reproduce. It is ending up with error AttributeError: module 'cgi' has no attribute 'escape'

**Comment by icarter09** [comment]
what version of Python are you running?

**Comment by icarter09** [comment]
@claudiux is this related to Python 3.2 deprecating `cgi.escape()` for `html.escape()` ?

**Comment by claudiux** [comment]
@icarter09 It seems to be that.

**Comment by icarter09** [comment]
#8956

**Comment by msjr65** [comment]
> @claudiux is this related to Python 3.2 deprecating `cgi.escape()` for `html.escape()` ?
> 
> #8959

Python Version is 3.8. Upgraded from 3.7 through regular pacman update on November 15 and indeed I have the impression that since then the issue started occuring.

**Comment by icarter09** [comment]
@eli-schwartz

**Comment by eli-schwartz** [comment]
This is fixed by 058e3a1e7dd26eca4bacb37b7c20c1e9d430de6a, which is not yet tagged and released.

**Comment by msjr65** [comment]
Thank you @eli-schwartz 
Fixed this myself for the time being by adjusting the .py files as outlined in  [058e3a1](https://github.com/linuxmint/cinnamon/commit/058e3a1e7dd26eca4bacb37b7c20c1e9d430de6a)
